docs/_index.md

```diff
@@ -282,42 +282,68 @@ To uninstall Poetry, simply delete the entire `$VENV_PATH` directory.
 `poetry` supports generating completion scripts for Bash, Fish, and Zsh.
 See `poetry help completions` for full details, but the gist is as simple as using one of the following:
 
+### Bash
+
+There are few options in `bash`, choose the one that suits your workflow.
+
+#### Local, recommended
 
 ```bash
-# Bash
-poetry completions bash > /etc/bash_completion.d/poetry
+poetry completions bash >> ~/.bash_completion
+```
 
-# Fish
-poetry completions fish > ~/.config/fish/completions/poetry.fish
+#### Local, based on XDG directory spec
 
-# Zsh
-poetry completions zsh > ~/.zfunc/_poetry
+```bash
+poetry completions bash > ${XDG_DATA_HOME:~/.local/share}/bash_completion/completions/poetry
+```
 
-# Oh-My-Zsh
-mkdir $ZSH_CUSTOM/plugins/poetry
-poetry completions zsh > $ZSH_CUSTOM/plugins/poetry/_poetry
+#### Global, might require root priviledges
```

Comment:
I would drop this entire section -- no point installing completions locally for a user-installed Poetry, and if Poetry is installed globally it's likely through distro packaging.

Reply:
removed